core/src/edu/asu/c3simulator/screens/FundingScreen.java

```diff
@@ -49,29 +50,27 @@ public void clicked(InputEvent event, float x, float y)
 				- (0.01f * stage.getWidth()), (0.01f * stage.getWidth()));
 		
 		// TODO add Corner Advisor
-		// TODO add Home Button
+		HomeButton home = new HomeButton(game);
 		
-		NavigationPanel navigationPanel = createNavigationPanel();
-		
-		stage.addActor(navigationPanel);
+		stage.addActor(home);
 		stage.addActor(continueButton);
 		stage.addActor(ScreenDiscription);
 	}
 	
 	@Override
-	public NavigationPanel createNavigationPanel()
+	public void createNavigationPanel()
```

Comment:
1: Lines 65-68 look like navigationPanel is accepting a Screen, not an AllManagementScreens instance
e.g.: navigationPanel.addButton("Industry", AllManagementScreens.INDUSTRY.getInstance());
AllManagementScreens.INDUSTRY.getInstance() is of type Screen

2: if createNavigationPanel **has exactly the same functionality as the "initialize" method**, then it should be renamed to initialize in order to be consistent with the rest of the code. 

2.1: AllManagementScreens.initialize() should only need to call the initialize() method of its screens, and nothing else. This will allow us to use a loop instead of hard coding. maintainability++ && size-- == true